src/UIKit/UIView.cs

```diff
@@ -176,6 +176,15 @@ public static Task<bool> AnimateAsync (double duration, NSAction animation)
 		{
 			return AnimateNotifyAsync (duration, animation);
 		}
+
+		public UIImage Capture (bool afterUpdates = false)
```

Comment:
`DrawViewHierarchy ` use `afterScreenUpdates` for it's parameter name. For consistency (with ourselves and the ObjC selector related to it) let's do the same.